.travis.yml

```diff
@@ -118,7 +130,7 @@ install:
 # command to run tests
 script:
   - 'if [ $SLYCOT != "" ]; then python -c "import slycot"; fi'
-  - coverage run setup.py test
+  - coverage run -m pytest --disable-warnings control/tests
```

Comment:
is `--disable-warnings` needed? I would rather have them printed and give every contributor the opportunity to fix problems early before e.g. `DeprecationWarnings` turn into errors after a Python or some library update.

Reply:
There are 18,209 warning messages and unfortunately pytest prints error messages first => I found it hard to locate the errors.  Most of the warnings seem to be around the use of `numpy.matrix`, which I figure we will get rid of starting in v0.9.0.